core/src/mindustry/ui/fragments/HudFragment.java

```diff
@@ -755,6 +806,16 @@ public void draw(){
             return builder;
         }).growX().pad(8f);
 
+        table.row();
+
+        table.table().update(t -> {
+            if(player.unit() instanceof Payloadc payload){
+                payload.contentInfo(t, 8 * 2, 280f);
```

Comment:
Updating the payload info **every frame** will lead to a lot of extra garbage. This should only happen when payloads are picked up/dropped.